tools/code-tools/jcstress.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# shellcheck disable=SC2035,SC2155
+set -euo pipefail
```

Comment:
Probably worth a one liner to say what this script is for (building multiple versions of jcstress)